SpywareFilter/sections/tracking_servers.txt

```diff
@@ -78,6 +78,7 @@
 ||tracdelight.io^$third-party
 ||goatcounter.com^$third-party
 ||gc.zgo.at^$third-party
+||tracker.idocdn.com:*/$third-party
```

Comment:
is `:*/` the necessary?

Reply:
Yes. AdGuard Apps.

Reply:
I changed it a little, but this is required `*`.

Reply:
Nope. I changed it.

Reply:
@Alex-302